versioned_docs/version-1.3.0/install-kubeslice/kubeslice-cli/topology-configuration.mdx

```diff
@@ -166,8 +167,9 @@ configuration:
     project_name: #{the name of the KubeSlice Project}
     project_users: #{optional: specify KubeSlice Project users with Read-Write access. Default is admin}
   helm_chart_configuration:
-    repo_alias: #{The alias of the helm repo for KubeSlice Charts}
+    repo_alias: #{The alias of the helm repo for KubeSlice Charts. For local charts provide the local path to the charts.}
```

Comment:
Add a comma after `For local charts, `

Reply:
Is it providing the local path of the charts? or `to the charts`?

Reply:
This is not fixed ....

Reply:
![image](https://github.com/kubeslice/docs/assets/105046156/25afcc4f-6566-4bba-b343-4bf34bb50913)

Reply:
this is fixed

Reply:
had fixed in the configuration parameters earlier. Now had fixed in the install doc

Reply:
Add a comma after `For local charts, ` 

This comment is not resolved